src/content/en/updates/2017/11/overscroll-behavior.md

```diff
@@ -0,0 +1,370 @@
+
+The property takes three possible values:
+
+1. **auto** - Default. Scrolls that originate on the element may propagate to
+ancestor elements. 
+- **contain** - prevents scroll chaining. Scrolls do not propagate to ancestors
+but local effects within the node are shown. For example, the overscroll glow
+effect on Android or the rubberbanding effect on iOS which notifies the user
+when they've hit a scroll boundary. **Note**: using
+`overscroll-behavior: contain` on the `html` element prevents overscroll
+navigation actions.
+- **none** - same as `contain` but it also prevents overscroll effects within
+the node itself (e.g. Android overscroll glow or iOS rubberbanding).
+
+Note: `overscroll-behavior` also supports shorthands for `overscroll-behavior-x`
+and `overscroll-behavior-y` if you only want to define behaviors for a certain
+axis.
+
+Let's dive into some examples to see how to use `overscroll-behavior`.
+
+## Prevent scrolls from escaping a fixed position element {: #fixedpos }
+
+### The chatbox scenario {: #chat }
+
+<figure class="attempt-right">
+  <a href="/web/updates/images/2017/11/overscroll-behavior/chatbox-chaining.mp4"
+     target="_blank">
+    <video
+      src="/web/updates/images/2017/11/overscroll-behavior/chatbox-chaining.mp4"
+      autoplay loop alt="Chatbox demo" height="350" class="border"></video>
+  </a>
+  <figcaption>Content beneath the chat window scrolls too :(</figcaption>
+</figure>
+
+Consider a fixed positioned chatbox that sits at the bottom of the page. The
+intention is that the chatbox is self-contained component and that it scrolls
+separately from the content behind it. However, because of scroll chaining, the
+document starts scrolling as soon as the user hits the last message in the chat
+history.
+
+For this app, it's more appropriate to have scrolls that originate within the
+chatbox stay within the chat. We can make that happen by adding
+`overscroll-behavior: contain` to the element that holds the chat messages:
+
+```css
+#chat .msgs {
+  overflow: auto;
+  overscroll-behavior: contain;
+  height: 300px;
+}
+```
+
+Essentially, we're creating a logical separation between the chatbox's scrolling
+context and the main page. The end result is that the main page stays put when
+the user reaches the top/bottom of the chat history. Scrolls that start in the
+chatbox do not propagate out.
+
+### The page overlay scenario {: #overlay }
+
+Another variation of the "underscroll" scenario is when you see content
+scrolling behind a **fixed position overlay**. A dead giveaway
+`overscroll-behavior` is in order! The browser is trying to be helpful but
+it ends up making the site look buggy.
+
+**Example** - modal with and without `overscroll-behavior: contain`:
+
+<figure class="flex">
+  <div class="flex1">
+    <a href="/web/updates/images/2017/11/overscroll-behavior/modal-off.mp4"
+       target="_blank">
+      <video src="/web/updates/images/2017/11/overscroll-behavior/modal-off.mp4"
```

Comment:
Just mp4, no webm?

Reply:
Do we care about that? Do other articles encode both formats?

Reply:
we usually do if it's not too much work, but not a hard requirement.